### ISSUE TITLE
TypeError: (0 , n.default) is not a function

### ISSUE THREAD

**Issue by MarkLyck** [marmelab/react-admin#3970]
After switching from using next.js to a simpler parcel/webpack build, I've been trying to solve this particular error for over a week to no avail.

I'm out of ideas how to solve this so I hope someone on here can help.

Full error:

```
TypeError: (0 , n.default) is not a function
    at admin.js:47
    at admin.js:23
    at Object.next (admin.js:4)
    at p (redux-saga-core.esm.js:1157)
    at yn (redux-saga-core.esm.js:1108)
    at redux-saga-core.esm.js:585
    at d (redux-saga-core.esm.js:56)
    at Y (redux-saga-core.esm.js:584)
    at redux-saga-core.esm.js:1204
    at h (redux-saga-core.esm.js:1271)
m @ io-6de156f3.js:111
```

After digging into this for quite a while I figured out this error happens on this line in the admin.ts file:

https://github.com/marmelab/react-admin/blob/master/packages/ra-core/src/sideEffect/admin.ts#L29

Now what's really weird is that it seems like it's importing html from that `./error` import instead of an actual JS module.

[![enter image description here][1]][1]

  [1]: https://i.stack.imgur.com/hGwrg.png

But that error is supposed to be a generator for some redux-saga stuff.

After switching to Parcel and being completely stumped by this error. I figured it might be some weird incompatibility between parcel and react-admin. So I switched out Parcel for a more typical webpack build... But the error persists through this. So it's likely nothing to do with the build tool.

Now to make things even more complex. If I manually go in and take out that error import and function call from the react-admin code.

a new error shows up `'o' is not a function` so it seems to me that there's something seriously screwed up.

I have not changed the version or implementation of react-admin at all.

This is using "react-admin": "^2.9.8",

I have no idea how this happened. I even tried redoing everything from scratch and I still ended up with the same error not touching the react-admin bits at all.

**Comment by Kmaschta** [comment]
Might be related to #3945 ?
We usualy redirect questions like that to Stack Overflow with the tag `react-admin`.

Do you think something in the codebase or in the documentation can be improved to avoid this situation?

**Comment by MarkLyck** [comment]
Well at first I thought it was an incompatibility with parcel-js, and the last issue was closed because "The core team don't use parcel," (which is why I went through the trouble to reproduce the error with webpack)

I've spent over a week trying to find the root of this error now, and I've ruled out parcel.js and redux-saga as the problem and narrowed down the issue to either a core problem with react-admin, or how we are using it. (in which I would expect there to be some kind of readable error happening) and not a random failed import of an `./error` module?

I already posted the same question on SO: https://stackoverflow.com/questions/58806461/react-admin-typeerror-0-n-default-is-not-a-function

But I created this issue since I believe this is likely a bug inside react-admin.
